extensions/job_management/standard/requirements/ogcapi-processes/create/REQ_body.adoc

```diff
@@ -0,0 +1,8 @@
+[[req_ogcapi-processes_create_body]]
+[requirement]
+====
+[%metadata]
+label:: /req/ogcapi-processes/create-body
+part:: The body of the POST request SHALL be based upon the OpenAPI 3.0 schema https://github.com/opengeospatial/ogcapi-processes/blob/master/openapi/schemas/processes-workflows/execute-workflows.yaml[execute-workflows.yaml]
+part:: The media type `application/json` SHALL be used to indicate that request body contains a processes description encoded as an <<rc_ogcapi-processes,OGC API - Processes>>.
```

Comment:
openEO also uses application/json... How do we expect to resolve such conflicts?

Reply:
See https://github.com/opengeospatial/ogcapi-processes/pull/437/files/7b79c502c609ee7ea352499a7fc521ab16551a65#r1790465176